metadata-ingestion/src/datahub/ingestion/source/redshift/redshift.py

```diff
@@ -592,7 +592,7 @@ def gen_table_dataset_workunits(
             table=table,
             database=database,
             schema=table.schema,
-            sub_type=table.type,
+            sub_type=DatasetSubTypes.TABLE,
```

Comment:
Are we already using the views subtype as expected?

Reply:
On the view side, it is DatasetSubTypes.View. Here in the past, I thought there is value to capture types from Redshift, but in the meantime, we went with the generic types